lib/view/multi-tasks-view.js

```diff
@@ -119,9 +120,10 @@ class MultiTasksView {
         this.taskRunner.run().then((context) => {
             callback(null, context);
         }).catch((listrError) => {
-            // listError { errors: [], context } contains array of errors from tasks and all context
+            const errorMessage = listrError.errors
+                .map(e => e.resultMessage || e.message || e).join('\n');
```

Comment:
taking care of all possible ways we throw the error. In this specific situation the error was on resultMessage